src/stylesheets/elements/form-controls/_date-picker.scss

```diff
@@ -55,6 +113,12 @@ $date-picker-max-width: "mobile";
   background-size: units(3);
   margin-top: 0.5em;
   width: 3em;
+
+  @media (forced-colors: active) {
+    @include add-color-icon($calendar, $contrast-bg);
+    height: auto;
+    margin: 1em 0.5em 0.5em;
```

Comment:
I can remove the height and margin and bump up the height of the mask, it's just going to be larger since it masks the whole button

![image](https://user-images.githubusercontent.com/25211650/159319323-d1c0c089-b07b-40bb-815b-e8590fa5a73f.png)

Thoughts?

Reply:
Resolved in [e1437af](https://github.com/uswds/uswds/pull/4581/commits/e1437afd5c2f544b092ac269acebdfd7f05e6b48)